<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUp</class>
 <widget class="QWidget" name="signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:white;
	border:2px solid black;
	border-radius:40px;
}</string>
  </property>
  <widget class="QGroupBox" name="gB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>700</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="signupBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	fontsize:200;
	border:none;
	background-color:rgb(28, 130, 255);
	border-radius:15px;
	color:black;
}
QPushButton:Hover
{
	fontsize:20;
	border:none;
	background-color:rgb(28, 130, 255);
	border-radius:15px;
	color:white;
}
QPushButton:Pressed
{
	fontsize:20;
	border:5px solid rgb(241, 92, 34);
	background-color:rgb(28, 130, 255);
	border-radius:15px;
	color:red;
}
</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>750</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mono</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid black;
    border-radius: 40px;
	margin-top:30px;
    font-size: 20px;
	font:Andale Mono;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 10px;
}</string>
    </property>
    <property name="title">
     <string>Registration information(required)</string>
    </property>
    <widget class="QLineEdit" name="password_Line_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	border: none;
	border-bottom: 3px solid rgb(241, 92, 34);
	background-color: Transparent;
	color: black;
	padding: 5px 5px 5px 5px;
}
QLineEditFocus
{
	border: 2px solid rgb(241, 92, 34);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_Line_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	border: none;
	border-bottom: 3px solid rgb(241, 92, 34);
	background-color: Transparent;
	color: black;
	padding: 5px 5px 5px 5px;
}
QLineEditFocus
{
	border: 2px solid rgb(241, 92, 34);
}</string>
     </property>
     <property name="placeholderText">
      <string>Enter Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-color:grey;
	border:none;
}</string>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-color:grey;
	border:none;
}</string>
     </property>
     <property name="text">
      <string>password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-color:grey;
	border:none;
}</string>
     </property>
     <property name="text">
      <string>identity</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_doc_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Doctor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_pat_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>200</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Patient</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-color:grey;
	border:none;
}</string>
     </property>
     <property name="text">
      <string>c.password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_Line2_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	border: none;
	border-bottom: 3px solid rgb(241, 92, 34);
	background-color: Transparent;
	color: black;
	padding: 5px 5px 5px 5px;
}
QLineEditFocus
{
	border: 2px solid rgb(241, 92, 34);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Comfirm Password</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:none;
	background-color:white;
	border-radius:20px;
}
QPushButton:Hover
{
	fontsize:20;
	border:none;
	background-color:rgb(28, 130, 255);
	border-radius:20px;
	color:red;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>750</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid black;
    border-radius: 40px;
	margin-top:30px;
    font-size: 20px;
	font:Andale Mono;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 10px;
}</string>
    </property>
    <property name="title">
     <string>Other information(optional)</string>
    </property>
    <widget class="QLineEdit" name="age_Line_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	border: none;
	border-bottom: 3px solid rgb(241, 92, 34);
	background-color: Transparent;
	color: black;
	padding: 5px 5px 5px 5px;
}
QLineEditFocus
{
	border: 2px solid rgb(241, 92, 34);
}</string>
     </property>
     <property name="placeholderText">
      <string>Enter Age</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="telephone_Line_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	border: none;
	border-bottom: 3px solid rgb(241, 92, 34);
	background-color: Transparent;
	color: black;
	padding: 5px 5px 5px 5px;
}
QLineEditFocus
{
	border: 2px solid rgb(241, 92, 34);
}</string>
     </property>
     <property name="placeholderText">
      <string>Enter Telephone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-color:grey;
	border:none;
}</string>
     </property>
     <property name="text">
      <string>age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-color:grey;
	border:none;
}</string>
     </property>
     <property name="text">
      <string>telephone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	font-color:grey;
	border:none;
}</string>
     </property>
     <property name="text">
      <string>gender</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_male_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>160</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cB_female_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>160</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
